CI is failing because the test is unable to include the header file:
```
CMakeFiles/RobustPredicateTestDriver.dir/predicates_test.cxx.o: In function
 `test(double, double, int)':
predicates_test.cxx:(.text+0x7d): undefined reference to `orient2d'
predicates_test.cxx:(.text+0x9a): undefined reference to `orient2dfast'
predicates_test.cxx:(.text+0xb0): undefined reference to `incirclefast'
```

Wondering whether the `extern "C"` should be removed and directly say `#include "predicates.h"`  in the test source file.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,5 @@
 cmake_minimum_required(VERSION 2.8.9)
 project(RobustPredicate)
-set(RobustPredicate_LIBRARIES RobustPredicate)
 
 if(NOT ITK_SOURCE_DIR)
   find_package(ITK REQUIRED)

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,10 +1,10 @@
-set(RobustPredicate_SRC
+set(${itk-module}_SRC
 predicates.c
 )
 
 add_library(${itk-module} ${${itk-module}_SRC})
 
-target_link_libraries(${itk-module}  ${${itk-module}_LIBRARIES})
+target_link_libraries(${itk-module} ${ITK_LIBRARIES})
 
 itk_module_target(${itk-module})
 

--- a/test/predicates_test.cxx
+++ b/test/predicates_test.cxx
@@ -1,6 +1,5 @@
-extern "C"{
 #include "predicates.h"
-}
+
 
 int test( double eps_x, double eps_y, int exact )
 {

